app.amsterdam.json

```diff
@@ -113,5 +113,8 @@
   },
   "sentry": {
     "dsn": "https://3de59e3a93034a348089131aa565bdf4@sentry.data.amsterdam.nl/27"
+  },
+  "azure": {
+    "connectionString": "InstrumentationKey=c39a786a-82ac-471d-a0c6-feb41eb4967b;IngestionEndpoint=https://westeurope-5.in.applicationinsights.azure.com/;LiveEndpoint=https://westeurope.livediagnostics.monitor.azure.com/"
```

Comment:
I would leave this empty, as we are not going to use `app.amsterdam.json` for deployments on Azure. There you can directly set the correct values in the Helm release.